docs/middleware/filesystem.md

```diff
@@ -1,300 +0,0 @@
----
-id: filesystem
----
-
-# FileSystem
```

Comment:
we also need to adjust the awesome list
https://github.com/gofiber/awesome-fiber?tab=readme-ov-file#-core